src/api/java/com/minecolonies/api/inventory/InventoryCitizen.java

```diff
@@ -41,6 +41,11 @@
      */
     private static final int ARMOR_SIZE = 5;
 
+    /**
+     * Amount of free slots
+     */
+    private int freeSlots = DEFAULT_INV_SIZE + ARMOR_SIZE;
+
```

Comment:
I guess loading the inv from NBT loads things nicely too? Not sure armor slots should be dealt with as free slots? We practically only use armor slots for rendering.

Reply:
ye checked it and entities now have their own seperate equipment/hands inventory, so removed this legacy code